<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowMenu</class>
 <widget class="QMainWindow" name="MainWindowMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>360</width>
    <height>497</height>
   </size>
  </property>
  <property name="sizeIncrement">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Blipper</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>331</width>
      <height>421</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QGroupBox" name="groupBoxCounter">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>280</height>
        </size>
       </property>
       <property name="title">
        <string>Counter</string>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
       <widget class="QWidget" name="formLayoutWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>311</width>
          <height>251</height>
         </rect>
        </property>
        <layout class="QFormLayout" name="formLayout">
         <property name="fieldGrowthPolicy">
          <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
         </property>
         <property name="rowWrapPolicy">
          <enum>QFormLayout::DontWrapRows</enum>
         </property>
         <property name="verticalSpacing">
          <number>6</number>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="labelCategory">
           <property name="text">
            <string>Category</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="comboBoxCategory">
           <item>
            <property name="text">
             <string>Process</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="labelCounter">
           <property name="text">
            <string>Counter</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QComboBox" name="comboBoxCounter">
           <item>
            <property name="text">
             <string>% Processor Time</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="labelProcessor">
           <property name="text">
            <string>Process</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QListWidget" name="listWidgetProcesses">
           <property name="maximumSize">
            <size>
             <width>250</width>
             <height>180</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <zorder>formLayoutWidget</zorder>
       <zorder>groupBoxCycle</zorder>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBoxCycle">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>110</height>
        </size>
       </property>
       <property name="title">
        <string>Cycle</string>
       </property>
       <widget class="QWidget" name="formLayoutWidget_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>311</width>
          <height>81</height>
         </rect>
        </property>
        <layout class="QFormLayout" name="formLayout_2">
         <property name="fieldGrowthPolicy">
          <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="labelUpdateInterval">
           <property name="text">
            <string>Update Interval</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QDoubleSpinBox" name="doubleSpinBoxInterval">
           <property name="suffix">
            <string> sec</string>
           </property>
           <property name="minimum">
            <double>0.250000000000000</double>
           </property>
           <property name="maximum">
            <double>5.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>0.250000000000000</double>
           </property>
           <property name="value">
            <double>1.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="labelRunTime">
           <property name="text">
            <string>Run Time</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QSpinBox" name="spinBoxRunTime">
           <property name="suffix">
            <string> sec</string>
           </property>
           <property name="minimum">
            <number>5</number>
           </property>
           <property name="maximum">
            <number>60</number>
           </property>
           <property name="singleStep">
            <number>5</number>
           </property>
           <property name="value">
            <number>5</number>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="labelProgressBar">
           <property name="text">
            <string>Progress</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QProgressBar" name="progressBarCycles">
           <property name="value">
            <number>24</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <zorder>formLayoutWidget_2</zorder>
       <zorder>listWidgetProcesses</zorder>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>360</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionGo"/>
    <addaction name="actionUpdate_Action_List"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionGo">
   <property name="text">
    <string>Go!</string>
   </property>
   <property name="toolTip">
    <string>Do it!</string>
   </property>
   <property name="shortcut">
    <string>Return</string>
   </property>
   <property name="priority">
    <enum>QAction::NormalPriority</enum>
   </property>
  </action>
  <action name="actionUpdate_Action_List">
   <property name="text">
    <string>Update Action List</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>progressBarCycles</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MainWindowMenu</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>227</x>
     <y>234</y>
    </hint>
    <hint type="destinationlabel">
     <x>182</x>
     <y>219</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
